src/components/RequesterPaysModal.js

```diff
@@ -44,42 +45,46 @@ const RequesterPaysModal = ({ onDismiss, onSuccess }) => {
     }, [
       spinnerOverlay
     ])],
-    [billingList.length > 0, () => h(Modal, {
-      title: 'Choose a billing project',
+    [workspaceList.length > 0, () => h(Modal, {
+      title: 'Choose a workspace to bill to',
       onDismiss,
       shouldCloseOnOverlayClick: false,
       okButton: h(ButtonPrimary, {
-        disabled: !selectedBilling,
+        disabled: !selectedGoogleProject,
         onClick: () => {
-          onSuccess(selectedBilling)
+          onSuccess(selectedGoogleProject)
         }
       }, ['Ok'])
     }, [
-      'This data is in a requester pays bucket. Choose a billing project to continue:',
+      'This data is in a requester pays bucket. Choose a workspace to bill to in order to continue:',
       h(IdContainer, [id => h(Fragment, [
-        h(FormLabel, { htmlFor: id, required: true }, ['Billing Project']),
+        h(FormLabel, { htmlFor: id, required: true }, ['Workspace']),
         h(Select, {
           id,
           isClearable: false,
-          value: selectedBilling,
-          placeholder: 'Select a billing project',
-          onChange: ({ value }) => setSelectedBilling(value),
-          options: _.uniq(_.map('projectName', billingList)).sort()
+          value: selectedGoogleProject,
+          placeholder: 'Select a workspace',
+          onChange: ({ value }) => setSelectedGoogleProject(value),
+          options: _.sortBy(option => option.label, _.uniq(_.map(workspace => {
+            return {
+              value: workspace.workspace.googleProject, label: `${workspace.workspace.namespace}/${workspace.workspace.name}`
+            }
+          }, workspaceList)))
```

Comment:
nit: I think it is good to switch to `flow` once we hit three nested functions for clarity - it can help future readers by not making them "unwrap" the code from the inside out mentally.

nit: On the map/destructure - I prefer the destructure because it makes it a bit more obvious that all of the values are coming from the same place in the object

```suggestion
          options: _.flow(
            _.map(({ workspace: { workspace: { googleProject, namespace, name } } }) => ({
                value: googleProject, label: `${namespace}/${name}`
            })),
            _.uniq,
            _.sortBy('label')
          )(workspaceList)
```